airbyte-integrations/connectors/source-shopify-singer/unit_tests/unit_test.py

```diff
@@ -22,6 +22,5 @@
 SOFTWARE.
 """
 
-
```

Comment:
unintentional?

Reply:
yes but formatting didn't change it so keeping it.

Reply:
this is breaking the build.

Reply:
fixing (hopefully) with this https://github.com/airbytehq/airbyte/pull/941

Reply:
`./gradlew build` didn't change it. I should have probably used `format`. thanks for fixing.

Reply:
not sure why but formatter locally wants to remove this line but then in ci it wants it. 🤷‍♀️ . looks like there's some other formatting that it's complaining about now... we can dig in more tomorrow.